pkg/operators/wasm/testdata/dataemit/gadget.yaml

```diff
@@ -0,0 +1,13 @@
+name: dataEmit
```

Comment:
This file in not necessary. I'd remove it.

Reply:
@blanquicet Done!